fabric-rendering-v1/src/main/java/net/fabricmc/fabric/mixin/client/rendering/MixinWorldRenderer.java

```diff
@@ -166,8 +167,44 @@ private void afterRender(CallbackInfo ci) {
 		WorldRenderEvents.END.invoker().onEnd(context);
 	}
 
-	@Inject(method = "Lnet/minecraft/client/render/WorldRenderer;reload()V", at = @At("HEAD"))
+	@Inject(method = "reload()V", at = @At("HEAD"))
 	private void onReload(CallbackInfo ci) {
 		InvalidateRenderStateCallback.EVENT.invoker().onInvalidate();
 	}
+
+	@Inject(at = @At("HEAD"), method = "renderWeather", cancellable = true)
+	private void renderWeather(LightmapTextureManager manager, float tickDelta, double x, double y, double z, CallbackInfo info) {
+		EnvironmentRenderers.WeatherRenderer renderer = null;
+
+		if (this.client.world != null) renderer = EnvironmentRenderers.getWeatherRenderer(world.getRegistryKey());
```

Comment:
Checkstyle wants you to always use brackets with if statements. Just giving you a heads up (I might be wrong, though)